src/main/resources/mappings/opendistro-ism-config.json

```diff
@@ -0,0 +1,321 @@
+{
+  "_doc": {
+    "properties": {
+      "dynamic": "strict",
+      "policy": {
+        "properties": {
+          "name": {
+            "type": "text",
+            "fields": {
+              "keyword": {
+                "type": "keyword",
+                "ignore_above": 256
+              }
+            }
+          },
+          "schema_version": {
+            "type": "long"
+          },
+          "last_updated_time": {
+            "type": "date",
+            "format": "strict_date_time||epoch_millis"
+          },
+          "default_state": {
+            "type": "keyword"
+          },
+          "states": {
+            "type": "nested",
+            "properties": {
+              "name": {
+                "type": "keyword"
+              },
+              "actions": {
+                "type": "nested",
+                "properties": {
+                  "timeout": {
+                    "type": "text"
+                  },
+                  "retry": {
+                    "properties": {
+                      "count": {
+                        "type": "integer"
+                      },
+                      "backoff": {
+                        "type": "keyword"
```

Comment:
What does `backoff` means, form text it looks like it should be boolean ?

Reply:
It's the type of backoff, such as exponential or linear. So it would look like this:

```
{
    // ... other action specific settings
    "timeout": "1h",
    "retry": {
        "count": 3,
        "backoff": "exponential",
        "delay": "5s"
    }
}
```